com.unity.render-pipelines.high-definition/Editor/Material/UIBlocks/SurfaceOptionUIBlock.cs

```diff
@@ -11,29 +11,54 @@
 
 namespace UnityEditor.Rendering.HighDefinition
 {
-    internal class SurfaceOptionUIBlock : MaterialUIBlock
+    /// <summary>
+    /// Surface Option properties in the material inspector.
+    /// </summary>
+    public class SurfaceOptionUIBlock : MaterialUIBlock
     {
+        /// <summary>
+        /// Surface Option features, allow you to hide certain parts of the UI.
```

Comment:
Options for surface option features. This allows you to display or hide certain parts f the UI.